go.mod

```diff
@@ -3,11 +3,17 @@ module github.com/drud/repo-chat-bot
 go 1.15
 
 require (
+	github.com/drud/admin-api v0.0.0-20201204132603-5134700cd97e
 	github.com/drud/cms-common v0.0.7
 	github.com/drud/ddev-live-sdk v0.2.8
 	github.com/whilp/git-urls v1.0.0
 	k8s.io/api v0.17.9
-	k8s.io/apimachinery v0.17.9
-	k8s.io/client-go v0.17.9
+	k8s.io/apimachinery v0.18.5
```

Comment:
admin-api brings this but then it replaces by v0.17.4. Likely because of another transitive dependency. We should probably clean up our go.mod files rather sooner than later.

https://github.com/drud/admin-api/blob/5134700cd97e73692837fd942581a608c48b8f52/go.mod#L26-L28